real-theta/options/videoStitching.js

```diff
@@ -0,0 +1,19 @@
+// COMMAND DEVELOPMENT STILL IN PROGRESS
```

Comment:
What do you need to complete?

Reply:
Based on the discussion just now and my testing, this code works just fine. But the camera must be in video mode prior to running this.